resources/translations/fr_MEL.json

```diff
@@ -93,9 +91,9 @@
   "mel.record.tab.map": "Cartographie",
   "mel.search.autocomplete.error": "Les suggestions ne peuvent pas être récupérLes suggestions ne peuvent pas être récupéréesées",
   "mel.search.field.any.placeholder": "Rechercher un jeu de données",
-  "mel.search.filter.generatedByWfs": "généré par une API",
   "mel.searchpage.subtitle.favorites": "Jeux de données suivis",
   "mel.tooltip.url.copy": "Copier l'URL racine",
+  "record.externalViewer.open": "",
```

Comment:
Could you copy the value from upstream here? This pops up now, since I must have forgotten to run the translations when I added the string.